environment.yml

```diff
@@ -13,7 +13,7 @@ dependencies:
   - matplotlib==3.6.2
   - mock==4.0.3
   - numpy==1.23.5
-  - pandas==1.2.3
```

Comment:
Are you sure this doesn't break anything? What version do you need for this script to work?

Reply:
Oops! That was me doing some local tests earlier this year and it slipped in. Adding versioning back in.